Content.Server/Chemistry/Containers/Components/SolutionContainerManagerComponent.cs

```diff
@@ -0,0 +1,20 @@
+using Content.Server.Chemistry.Containers.EntitySystems;
+using Content.Shared.Chemistry.Containers.Components;
+using Content.Shared.Chemistry.Solutions;
+
+namespace Content.Server.Chemistry.Containers.Components;
+
+/// <summary>
+/// Component used to spawn solution entities on <see cref="MapInitEvent"/>.
+/// Replaces itself with a <see cref="SolutionContainerComponent"/> during map initialization.
+/// </summary>
+[RegisterComponent]
+[Access(typeof(SolutionContainerSystem))]
```

Comment:
```suggestion
[Access(typeof(SolutionContainerSystem), Other = AccessPermissions.None)]
```
to avoid people accidentally using the wrong component.

Reply:
Actually, why aren't the two components just the same component? you can just put the access restriction on the "prototype" field. This would also significantly reduce the number of changes.

Reply:
It's because I was assuming it would be better for perf to deallocate the prototype dictionary when we're done with it so we don't have ~twice as much memory persistently allocated for each solution container.

Reply:
Made them the name comp